install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -45,6 +45,9 @@ spec:
           args:
           {{- if .Values.global.sds.enabled }}
             - --sds-enabled=true
+          {{- end }}
+          {{- if .Values.global.sds.useNormalJwt }}
```

Comment:
You mean if useTrustworthyJwt == false then allow legacy jwts?

Reply:
useTrustworthyJwt basically configs the pod to mount istio-token and other things for trustworthy jwt usage. If useNormalJwt set to true, envoy will fetch normal k8s service account JWT from '/var/run/secrets/kubernetes.io/serviceaccount/token'. So basically what this PR is doing is, by default, when SDS is enabled, use trustworthy jwt from the mounted istio token on proxy. A user can configure their Citadel to allow legacy jwts by setting `userNormalJwt` to true.

Reply:
Makes sense to me. Thanks.
Ideally we should use an enum for this, because useNormalJwt and useTrustworthyJwt are mutually exclusive. Let's defer that work and optimize it in istio/installer.

Reply:
I agree with the enum option.